t/perf/benchmarks

```diff
@@ -2191,6 +2191,12 @@
         code    => '$y = 0; $x = 2; out: $x--; while ($x) { goto out if $y++ > 4; redo; }',
     },
 
+    'loop::for::last4' => {
+        desc    => 'for loop containing for loop with a last',
+        setup   => '',
+        code    => 'for (1..4) { for (1..2) { last; } }',
+    },
```

Comment:
No real need for the outer 1..4 for loop. Porting/bench.pl will wrap the code in a for loop anyway.